02.general/17.dns-providers/docs.en.md

```diff
@@ -328,7 +333,7 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 
 <a name="opennic-dns"></a>
 
-### OpenNIC DNS
+### OpenNIC DNSdnsovertls.sinodun.com
```

Comment:
You should better use `OpenNIC DNS: dnsovertls.sinodun.com` or `OpenNIC DNS (dnsovertls.sinodun.com)` instead of `OpenNIC DNSdnsovertls.sinodun.com`. Correct it, please.